test/functional/sendheaders.py

```diff
@@ -263,7 +266,10 @@ def test_null_locators(self, test_node):
         test_node.send_get_headers(locator=[], hashstop=int(block.hash, 16))
         test_node.sync_with_ping()
         assert_equal(test_node.block_announced, False)
+        inv_node.clear_last_announcement()
         test_node.send_message(msg_block(block))
+        inv_node.check_last_announcement(inv=[int(block.hash, 16)], headers=[])
+        inv_node.clear_last_announcement()
```

Comment:
nit: is this required? check_last_announcement already includes the code of clear_last_announcement

Reply:
not required. Now removed